assignment_2.rb

```diff
@@ -0,0 +1,78 @@
+class Company
+ attr_accessor :company_id,:company_name,:no_of_employee,:no_of_division,:hr_head_name
+ def initialize(company_id,company_name,no_of_employee,no_of_division,hr_head_name)
+  @company_id=company_id
+  @company_name=company_name
+  @no_of_employee=no_of_employee
+  @no_of_division=no_of_division
+  @hr_head_name=hr_head_name
+ end
+
+ def company_details()
+  puts "The name of the company is #{@company_name} and the number of employee #{@no_of_employee} and the number of division is #{@no_of_division}"
+ end
+
+ private
+  def hr_details()
```

Comment:
Do not use () if you are not passing any arguments.

Reply:
done